esp/src/eclwatch/XrefQueryWidget.js

```diff
@@ -127,6 +127,7 @@ define([
                     }
                 });
 
+                ESPUtil.goToPageUserPreference(retVal, "XrefQueryWidget_GridRowsPerPage");
```

Comment:
This one may need testing...

Reply:
You need to check where its being called from (as its in a function) and possibly return a promise from this function?